test/unittest/security/logging/LoggingPluginTests.hpp

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 Canonical, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef _UNITTEST_SECURITY_LOGGING_LOGGINGPLUGINTESTS_HPP_
+#define _UNITTEST_SECURITY_LOGGING_LOGGINGPLUGINTESTS_HPP_
+
+#include "../../../../src/cpp/fastrtps_deprecated/security/logging/LogTopic.h"
+#include "../../../../include/fastdds/rtps/attributes/PropertyPolicy.h"
```

Comment:
Yick. Why not add an include dir in the test's CMakeLists.txt?

Reply:
Just following what's been established so far.

Reply:
Okay, if they do this elsewhere then fair enough.